src/Worker/ExpireAndRemoveUsers.php

```diff
@@ -34,8 +35,8 @@ class ExpireAndRemoveUsers
 	public static function execute()
 	{
 		// expire any expired regular accounts. Don't expire forums.
-		$condition = ["NOT `account_expired` AND `account_expires_on` > ? AND `account_expires_on` < UTC_TIMESTAMP() AND `page-flags` = ? AND `uid` != ?",
-			DBA::NULL_DATETIME, User::PAGE_FLAGS_NORMAL, 0];
+		$condition = ["NOT `account_expired` AND `account_expires_on` BETWEEN ? AND ? AND `page-flags` = ? AND `uid` != ?",
```

Comment:
`BETWEEN` is including the boundary values. So this command would expire all accounts.

Reply:
Ha! Thanks for catching it.